KoreBuild-dotnet/build/_dotnet-test.shade

```diff
@@ -18,7 +18,16 @@ default configuration = 'Debug'
 
 @{
     var projectFolder = Path.GetDirectoryName(projectFile);
```

Comment:
My guess is that `dotnet test` is a general purpose kick starter for a test runner, so you could have something like NUnit which might not support parallelism in which case this argument does not hold true.